extensions/wikidata/src/org/openrefine/wikidata/schema/WbDateConstant.java

```diff
@@ -119,7 +119,7 @@ public static TimeValue parse(String datestamp)
             calendar.setTime(date);
             return Datamodel.makeTimeValue(calendar.get(Calendar.YEAR), (byte) (calendar.get(Calendar.MONTH) + 1), 
                     (byte) calendar.get(Calendar.DAY_OF_MONTH), (byte) calendar.get(Calendar.HOUR_OF_DAY),
-                    (byte) calendar.get(Calendar.MINUTE), (byte) calendar.get(Calendar.SECOND), (byte) precision, 0, 1,
```

Comment:
what's the implication of this change? Maybe add some comments since Datamodel is from wikidata.

Reply:
the implication is that now OpenRefine and the Wikidata UI will agree on the default value for this "after" parameter. I can elaborate here but I am not sure how useful it would be to add more comments in the code itself, given that it's just one of the default values (with the "before" and "timezone" parameters) - having a default value of 0 is fairly intuitive, no?